frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -1,10 +1,7 @@
 import type { ReactNode } from "react";
 import { useCallback, useMemo } from "react";
 
-import {
-  getColumnGroupIcon,
-  getColumnGroupName,
-} from "metabase/common/utils/column-groups";
+import { getColumnGroupIcon } from "metabase/common/utils/column-groups";
```

Comment:
`getColumnGroupIcon` didn't make it to `Lib.filterGroups` because it references `IconName` from `metabase/ui` which is a forbidden import.